streaming/index.js

```diff
@@ -512,7 +516,8 @@ const startWorker = async (workerId) => {
    * @param {any} res
    */
   const subscribeHttpToSystemChannel = (req, res) => {
-    const systemChannelId = `timeline:access_token:${req.accessTokenId}`;
+    const accessTokenChannelId = `timeline:access_token:${req.accessTokenId}`;
+    const systemChannelId = `timeline:token:${req.accountId}`;
```

Comment:
Naming it `token` and then using `accountId` is not very intuitive. I suppose this has got to be something like `timeline:${req.accountId}:system` to be consistent with other channel names (e.g. `timeline:${req.accountId}:notifications`)

Reply:
That was an error on my part! I intended to name that channel `timeline:system:${req.accountId}`! Fixed it.

Wonder about `timeline:${req.accountId}:system` VS `timeline:system:${req.accountId}` though